### DISCUSSION TITLE
Add a 'Request Link Confirmation' option

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#1422] [Ideas]
I don't know if there's a crowd for this feature besides me, but I'd love an optional setting like ProtonMail's 'Request Link Confirmation', where if turned on, there would be a prompt with the full URL of a link that you'd need to accept before opening the internal/external browser.

Sometimes I don't realize that a picture for example is a thumbnail of a link and when I click it opens my browser. This would help for instances like that and it would save you if you accidentally clicked on a malicious link from some weird Subreddit.

Again I don't know if there's a crowd for this, but if I had a project like this, I'd definitely add it. If it's easy and worth it we could have on the prompt a 'don't ask again for this domain' checkbox, where if clicked the domain would automatically open without confirming again and again.

**Comment by Gitchick57** [comment]
There is a need for this.  I uninstalled the Reddit native app because they were "quietly" redirecting me to Google Chrome to open links when Google Chrome or search engine is not my default app / search. 



Mar 28, 2023, 12:01 PM by ***@***.***:
